File: AppGR/FontEnd/app/src/main/java/com/example/appgrit/models/DeletePostModel.java
package com.example.appgrit.models;

public class DeletePostModel {
    private String UserId;
    private String PostId;

    // Constructors, getters, and setters

    public DeletePostModel() {
    }

    public DeletePostModel(String userId, String postId) {
        UserId = userId;
        PostId = postId;
    }

    public String getUserId() {
        return UserId;
    }

    public void setUserId(String userId) {
        UserId = userId;
    }

    public String getPostId() {
        return PostId;
    }

    public void setPostId(String postId) {
        PostId = postId;
    }
}

File: AppGR/BackEnd/AppGrIT/Models/DeletePostModel.cs
using System.ComponentModel.DataAnnotations;

namespace AppGrIT.Models
{
    public class DeletePostModel
    {
        [Required]
        public string PostId { get; set; } = null!;
        [Required]
        public string UserId
        { get; set;} = null!;


    }
}
